core/protocol.ml

```diff
@@ -76,8 +76,8 @@ module In = struct
     | Get of string list         (** return a web interface resource *)
   [@@deriving rpc]
 
-  let slash = Re_str.regexp_string "/"
-  let split = Re_str.split_delim slash
+  let slash = Re.Str.regexp_string "/"
+  let split = Re.Str.split_delim slash
```

Comment:
I know it was like this before, but this is not thread-safe, it uses a global variable, see https://github.com/ocaml/ocaml-re/blob/159c491687a4a61f432295ed1477675d311c695b/lib/str.ml#L35
We should use either Re.Posix (and Re.split), or a function from Astring.String to do the splitting.